### PR TITLE
[CSL-1778] Set termSeverity for demo.sh

### PR DESCRIPTION
**Pull request by int-index**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -64,6 +64,11 @@ packages:
     git: https://github.com/input-output-hk/cardano-crypto
     commit: 1cde8e3a8d9093bbf571085920045c05edb3eaa4
   extra-dep: true
+- location:
+    git: https://github.com/serokell/log-warper
+    commit: a2db552a9ce0fd58d82e3d1ffd147a8474d8db5c # master
+  extra-dep: true
```
Comment (gromakovsky) [inline stack.yaml]: Let's use Hackage version, I am going to upload it shortly.
And CI failed because you didn't update default.nix.

Comment (gromakovsky) [reply stack.yaml]: https://github.com/serokell/log-warper/pull/30

Comment (gromakovsky) [reply stack.yaml]: https://hackage.haskell.org/package/log-warper

**Comment by int-index** [comment]
Ok, I created #1791

**Pull request merged**